view/frontend/web/js/view/payment/method-renderer/adyen-pm-method.js

```diff
@@ -501,233 +501,220 @@ define(
-                            addressLine2: formattedShippingAddress.houseNumber,
-                            city: formattedShippingAddress.city,
-                            postalCode: formattedShippingAddress.postalCode,
-                            countryCode: formattedShippingAddress.country,
-                            phoneNumber: formattedShippingAddress.telephone
-                        };
-                        if (configuration.addressDetails.countryCode === 'US') {
-                            configuration.addressDetails.stateOrRegion = quote.shippingAddress().regionCode
-                        }
-                    }
-                    else if (formattedBillingAddress &&
-                        formattedBillingAddress.telephone) {
-                        configuration.addressDetails = {
-                            name: formattedBillingAddress.firstName +
-                                ' ' +
-                                formattedBillingAddress.lastName,
-                            addressLine1: formattedBillingAddress.street,
-                            addressLine2: formattedBillingAddress.houseNumber,
-
-                            city: formattedBillingAddress.city,
-                            postalCode: formattedBillingAddress.postalCode,
-                            countryCode: formattedBillingAddress.country,
-                            phoneNumber: formattedBillingAddress.telephone
-                        };
-                        if (configuration.addressDetails.countryCode === 'US') {
-                            configuration.addressDetails.stateOrRegion = quote.billingAddress().regionCode
-                        }
-                    }
                 }
 
-                if (paymentMethod.methodIdentifier.includes('affirm')) {
-                    configuration.visibility = {
-                        personalDetails: "hidden",
-                        billingAddress: "hidden",
-                        deliveryAddress: "hidden"
-                    }
-                }
+                let street = getStreetAndHouseNumberFromAddress(
+                    address,
+                    adyenConfiguration.getHouseNumberStreetLine(),
+                    adyenConfiguration.getCustomerStreetLinesEnabled()
+                );
 
-                return configuration;
+                return {
+                    city: address.city,
+                    country: address.countryId,
+                    postalCode: address.postcode,
+                    street: street.streetName,
+                    houseNumber: street.houseNumber,
+                    firstName: address.firstname,
+                    lastName: address.lastname,
+                    telephone: address.telephone
+                };
             },
+
+            //TODO when we have all individual paymentmethods we can extrapolate shared information to seperate file and delete below.
```

Comment:
Can we remove all this commented out code?

Reply:
I would personally wait until we have all payment methods moved to individual so that we have the necessary information for them.